### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by chickdan** [jellyfin/Swiftfin#879] [Discussions]
I saw the call for developers on Reddit and HackerNews but noticed this repo doesn't have the topic.

I've been fairly irked by the direction Plex is headed and seeing that Jellyfin has a native iOS client has me motivated to help out. I plan on doing what I can when I can, even after Hacktoberfest, but it would be nice to have it linked since I'm making the contributions anyway 🙂

**Comment by LePips** [comment]
I have added the tag, thank you for your contributions!

**Reply by chickdan** [reply]
Thank you!
